spec/integration/master_process_spec.rb

```diff
@@ -16,11 +16,20 @@
 
   let(:content_id) { 'id1' }
   let(:base_path) { '/the-base-path' }
+  let(:locale) { 'en' }
   let(:item_content) { 'This is the content.' }
 
-  let!(:an_item) { create :dimensions_item, content_id: 'a-content-id' }
-  let!(:item) { create :dimensions_item, content_id: content_id, base_path: base_path, latest: false }
-  let!(:outdated_item) { create :outdated_item, content_id: content_id, base_path: base_path }
+
+  let!(:an_item) { create :dimensions_item, content_id: 'a-content-id', locale: locale }
+  let!(:outdated_item) do
```

Comment:
It seems that during the `git rebase` we have lost the changes [of this commit](https://github.com/alphagov/content-performance-manager/pull/605/commits/9f6e2332501344b4060c676ad3bf2990e3681403)